### DISCUSSION TITLE
Bug Hunt-March 2021

### DISCUSSION THREAD

**Discussion by Accio1** [theforumhelpers/theforumhelpers.github.io#92] [Announcements]
### Welcome to the Forum Helpers March 2021 Bug Hunt!

**What is the Bug Hunt?**
The bug hunt is a new idea that I am trying out for this organisation. Basically, during the contest window, people look for bugs or inconsistencies across the website that have not already been reported, and report them here in the comments. After the bug hunt is over, the bugs are sorted and compiled into various issues which can then be resolved in order to improve the site!

**Rules**
- Bugs can be big or small, although generally larger bugs have been reported already, so it is smaller bugs that get reported.
- It doesn't necessarily have to be a bug. It could also be an inconsistency or something that just doesn't seem to fit with the rest of the site's code. For example, as of writing this, Helvetica is a part of the font family for a few pages, but it is not rendered on those pages because it wasn't imported.
- Report bugs in the comments of this discussion.
- Only report bugs that have not been reported/recorded elsewhere.
- This bug hunt starts when this is posted, and ends March 15th 2020 at 11:59PM

**Winner**
- There are two ways to win:
  - Discover the most bugs
  - Discover the most critical/high priority bug
- It is mostly just for fun, but the winners will get their names recorded in this post

Good Luck! Have fun!

Tagging people who have somewhat recently participated in this repository (you are in no way obligated to participate, I just thought that it would be a fun competition):
@leahcimto @The64thCucumber @jeffalo @MTM828 @awesome-llama @shefwerld

@leahcimto Congratulations on winning this Bug Hunt with two bugs reported during the timeframe!

**Comment by leahcimto** [comment]
QuickReply has no footer

**Comment by leahcimto** [comment]
The TOC on the list of forum helpers page has no bottom margin, though it has a top margin.

**Comment by ghost** [comment]
I'm not really sure if this counts or not, but the nav blocks on QuickReply and List Of Forum Helpers have different sizes due to the size of the links inside varying, they should (and not must) have the same size. The one on List Of Forum Helpers is like 4X bigger than the one on QuickReply. But again, it's not really a "bug".

**Comment by Accio1** [comment]
Thanks to @MTM828 and @leahcimto for submitting bug reports. With 2 bugs reported during the timeframe, @leahcimto is the winner of this Bug Hunt. Congratulations!

**Reply by leahcimto** [reply]
yay
